static/js/compose.js

```diff
@@ -536,6 +542,30 @@ export function initialize() {
         },
     );
 
+    $("#compose_pm_mentions_user_alert").on(
+        "click",
+        ".compose_pm_mentions_user_alert_close",
+        (event) => {
+            const stream_alert_row = $(event.target).parents(".compose_pm_mentions_user_alert");
+            const stream_alert = $("#compose_pm_mentions_user_alert");
+
+            stream_alert_row.remove();
+
+            if (stream_alert.children().length === 0) {
+                stream_alert.hide();
+            }
+        },
+    );
+    $("#compose_pm_mentions_user_alert").on("click", ".undo", (event) => {
```

Comment:
The `Undo` button is not working now because you are using incorrect class here. I think we can directly use the id of that button.

Reply:
done.